src/components/Header.vue

```diff
@@ -148,47 +148,49 @@ export default defineComponent({
 
 <template lang="pug">
 .header-background
-  .row.text-center.q-pt-sm.justify-between
-    .logo-container.col-xs-2.col-sm-2.col-md-2.col-lg-2.q-pa-xs-sm.q-pa-sm-xs.q-pa-md-md.q-pa-lg-md
-      a( href="/").float-left.q-ml-sm
-        img.logo( v-if="isSmall" src="~assets/telos_logo.svg")
-        img.logo-tlos( v-else src="~assets/tlos.png")
-    .col-xs-6.col-sm-6.col-md-4.col-lg-6.q-pa-xs-sm.q-pa-sm-xs.q-pa-md-md.q-pa-lg-md.q-pt-sm
-      .row.justify-center.full-width
-        q-select.col-12.search-input.q-pl-md(
-          borderless 
-          dense 
-          filled
-          :model-value="selected"
-          label-color="white"  
-          color="primary"
-          use-input
-          hide-selected
-          fill-input
-          input-debounce="0"
-          :options="options"
-          @update="executeSearch"
-          @keyup.enter="executeSearch" 
-          :input-style="{ color: 'white' }"
-          @click="executeSearch"
-          @input-value="getOptions")
-            template( v-slot:option="scope")
-              q-item(v-if="!scope.opt.groupLabel"
-                v-bind="scope.itemProps"
-                v-on="scope.itemEvents"
-                @click="suggestedSearch(scope.opt)"
-              )             
-                q-item-section
-                  q-item-label(v-html="scope.opt.label")
-              q-item(v-if="scope.opt.groupLabel"
+  .row.text-center.q-pt-sm.justify-between.q-pt-md
+    .logo-container.col-xs-2.col-sm-2.col-md-2.col-lg-2
+      .q-px-xs-xs.q-px-sm-xs.q-px-md-md.q-px-lg-md
+        a( href="/").float-left.q-ml-sm
+          img.logo( v-if="isSmall" src="~assets/telos_logo.svg")
+          img.logo-tlos( v-else src="~assets/tlos.png")
+    .col-xs-5.col-sm-6.col-md-4.col-lg-6
+      .q-px-xs-xs.q-px-sm-xs.q-px-md-md.q-px-lg-md
+        .row.justify-center.full-width
+          q-select.col-12.search-input(
+            borderless 
+            dense 
+            filled
+            :model-value="selected"
+            label-color="white"  
+            color="primary"
+            use-input
+            hide-selected
+            fill-input
+            input-debounce="0"
+            :options="options"
+            @update="executeSearch"
+            @keyup.enter="executeSearch" 
+            :input-style="{ color: 'white' }"
+            @click="executeSearch"
+            @input-value="getOptions")
+              template( v-slot:option="scope")
+                q-item(v-if="!scope.opt.groupLabel"
```

Comment:
you can re-order `q-item` s here for most likely case first as` v-if=<condition>`  and second as `v-else`

Reply:
Do you mean have a localstorage entry that saves past searches and order those on top?